src/modules/form/hooks/usePreloadPicklists.tsx

```diff
@@ -0,0 +1,79 @@
+import { useLazyQuery } from '@apollo/client';
+import { compact, isEmpty } from 'lodash-es';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+import { ItemMap } from '../types';
+
+import useStabilizedValue from '@/hooks/useStabilizedValue';
+import {
+  GetPickListDocument,
+  PickListOptionFieldsFragment,
+  PickListType,
+} from '@/types/gqlTypes';
+
+const usePreloadPicklists = (
+  itemMap: ItemMap | undefined,
+  relationId?: string
+) => {
+  const [fetchPickList] = useLazyQuery(GetPickListDocument);
```

Comment:
We should still use a network policy that re-fetches the pick lists because their values may have changed (if its an enrollment list or project coc list for example). Previously we were using network-only, but maybe cache-and-network here is more appropriate?

Reply:
I switched to using `client.query` since `useLazyQuery` wasn't working correctly. I'm using `network-only` as the fetch policy to ensure that the picklists are always fetched from network when we load a form.